### PR TITLE
This is to push a 'Announcement Input Box' page.

### PR DESCRIPTION
**Pull request by AnitaTT**
Set the Welcome page as "Home" page, on top of the Welcome page, created 2 pages:
1. "Say_hello" page
2. "Message_Index" page.

Functions in the Message_Index Page:
1. create a new announcement with 2 input boxes, one is "Message Title", and the other is " Message Content".
2. show details of an announcement.
3. edit details of an announcement with the 2 same input boxes and update them to the index.

### CODE REVIEW

**Review by ChiuWeiYi — COMMENTED** [review]
generally ok with two suggestions 

1) refactoring the controller, can refer to the D.R.Y principle.

2) the name of column could be better. Can refer to MySQL styling 

looking forward to seeing slack features soon!

File: slack-announcement-scheduler/db/migrate/20221002090614_create_messages.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateMessages < ActiveRecord::Migration[7.0]
+  def change
+    create_table :messages do |t|
+      t.text :name
+      t.text :description
+      t.text :department
+      t.text :when_to_announce
```
Comment (ChihHaoTW) [inline slack-announcement-scheduler/db/migrate/20221002090614_create_messages.rb:7]: Suggest using datetime to store time format
```suggestion
      t.datetime :when_to_announce
```

**Review by ChihHaoTW — CHANGES_REQUESTED** [review]
Got one suggestion, others look good to me!!

File: slack-announcement-scheduler/db/migrate/20221002090614_create_messages.rb (hunk 1 above)
Comment (ChihHaoTW) [reply slack-announcement-scheduler/db/migrate/20221002090614_create_messages.rb:7]: And agree with WY
This should be named like `announce_time`

**Review by ChihHaoTW — APPROVED** [review]

**Review by ChiuWeiYi — APPROVED** [review]

**Pull request merged**